app/repositories/SessionCacheRepository.scala

```diff
@@ -56,6 +56,12 @@ class DefaultSessionCacheRepository @Inject()(mongoComponent: MongoComponent,
     } yield result
   }
 
+  override def verifySessionId(sessionId: String): Future[Boolean] = {
+    for {
+      record <- collection.find(equal("_id", sessionId)).toSingle().toFutureOption()
+    } yield record.isEmpty
```

Comment:
Please use either record.isDefined or record.nonEmpty as return type of verifySessionId method is not conveying the right output as per name. The case statements in AccountLinkController.getSessionId should also be updated accordingly

Reply:
This has been addressed. hence resolving